proxy/src/telemetry/metrics/mod.rs

```diff
@@ -65,6 +66,15 @@ use self::labels::{
 pub use self::labels::DstLabels;
 pub use self::record::Record;
 
+trait FmtMetric {
```

Comment:
Since this trait seems kind of important, it would be nice to have a comment explaining why we have it instead of just using `fmt::Display`.